rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  def initialize(date, person, book)
+    @date = date
+
+    @person = person
+    person.rental << self
+
+    @book = book
+    book.rental << self
+  end
+
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] Similarly, `book` and `person` should be marked with `attr_accessor`, as we intend to both _access (get)_ as well as _write (set)_ to it. Kindly ensure to improve upon this aspect.